io.github.igorlogius.scr2ppm.yml

```diff
@@ -0,0 +1,27 @@
+app-id: io.github.igorlogius.scr2ppm
```

Comment:
Ideally, it should provide some actual UI (for example written in Python, and just executing src2ppm), but it's just a general comment.

Reply:
Noted, but i think for now i'll keep it as a mostly command line program ... i might change that maybe in a future version.